types/feathersjs__feathers/feathersjs__feathers-tests.ts

```diff
@@ -9,7 +9,7 @@ interface Services {
     users: User;
 }
 
-const app = feathers() as Application<Services>;
```

Comment:
We generally encourage using an explicit type assertion because it makes the intention clearer, but I can also see the appeal of the shorter expression.